.github/actions/get_dfx_version/action.yml

```diff
@@ -0,0 +1,16 @@
+name: Get dfx version
+description: Determines Azle's dfx version
+outputs:
+    dfx-version:
+        description: Returns the version of dfx Azle used for it's last template
```

Comment:
description: "Returns the version of dfx that Azle will test against and use to generate its Wasm binary template"

Reply:
Done